app-text/asciidoc/asciidoc-9999.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
I would agree, except that in this case the commits before this one were all authored in 2015 :-) .